utils/grpc/runtime_test.go

```diff
@@ -0,0 +1,51 @@
+package grpc
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"github.com/tron-us/go-btfs-common/config"
+	"github.com/tron-us/go-btfs-common/protos/shared"
+	"github.com/tron-us/go-common/v2/constant"
+	"github.com/tron-us/go-common/v2/log"
+	"go.uber.org/zap"
+	"strings"
+	"testing"
+)
+
+func TestRuntimeServer(t *testing.T) {
+	config.InitDB()
```

Comment:
can put this in `init()` so it's shared by all tests

Reply:
ok